frontend/sw360-portlet/src/main/webapp/html/components/edit.jsp

```diff
@@ -140,19 +140,52 @@
 
 require(['jquery', 'modules/sw360Validate', 'modules/autocomplete', 'modules/confirm' ], function($, sw360Validate, autocomplete, confirm) {
 
+    Liferay.on('allPortletsReady',
+        function () {
+            load();
```

Comment:
I am not the biggest JS expert, but is is necessary to nest load() in an anonymous function?

Reply:
Moved all load methods to the inline.